ert_shared/ensemble_evaluator/ensemble/prefect.py

```diff
@@ -53,7 +53,7 @@ async def _eq_submit_job(self, script_filename):
 
 
 def _get_executor(custom_port_range, name="local"):
-    _, port = find_available_port(custom_range=custom_port_range)
+    _, port, _ = find_available_port(custom_range=custom_port_range)
```

Comment:
Is the socket closed here when the it goes out of scope? :thinking: If not I am a bit surprised as to how this works, since the dask scheduler will try to bind to this port later.